### ISSUE TITLE
Incomplete Heuristic/Benchmarking.py module

### ISSUE THREAD

**Issue by sajith** [atlanticwave-sdx/pce#36]
It is unclear what [src/Heuristic/Benchmarking.py](https://github.com/atlanticwave-sdx/pce/blob/95ce66373e689db9b05daebb101c04ec8b62eafc/src/Heuristic/Benchmarking.py) does, and it doesn't seem that it is actually usable:

```shell
$ python3 src/Heuristic/Benchmarking.py 
Traceback (most recent call last):
  File "/home/sajith/projects/renci/aw-sdx/pce/src/Heuristic/Benchmarking.py", line 1, in <module>
    from solver import solvermethod
ImportError: cannot import name 'solvermethod' from 'solver' (unknown location)
```

Another error from flake8:

```shell
$ flake8 --count --show-source --select=E9,F63,F7,F82 src/Heuristic/Benchmarking.py
src/Heuristic/Benchmarking.py:147:22: F821 undefined name 'nxgraphgenerator'
            answer = nxgraphgenerator(nodes[0]+i,0.2,1000,20)
                     ^
```

@YufengXin @yifei666 What should we do with this module?  Could we correct the errors, and document it? Or could we remove it, or maybe at least move it out of `src` directory?

**Comment by yifei666** [comment]
Hi, this file is only used to generate the benchmark result for the computation time and plot the graph. Since we changed the return output of the Solver to be [solution, objective], this file would not work. This file required the solver to return [objective value, computation time] and it will be able to run and plot.

I kept this file in the repo so we don't need to write the plot script again if we want to do the benchmark in the future and it shows the workflow of the testing. It can be moved to utility folder.
